### PR TITLE
Added new fixer PHPDoc Scalar Full

### PR DESCRIPTION
**Pull request by rokerkony**
It is opposite solution for opposite to PHPDoc Scalar fixer.

Inspired by `PhpdocScalarFixer` Graham Campbell <graham@mineuk.com>

### CODE REVIEW

File: Symfony/CS/Fixer/Contrib/PhpdocScalarFullFixer.php [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+        if (strpos($usefulContent, '|') !== false) {
+            $newContent = implode('|', self::normalizeTypes(explode('|', $usefulContent)));
+        } else {
+            $newContent = self::normalizeType($usefulContent);
+        }
+
+        if ($newContent !== $usefulContent) {
+            // limiting to 1 replacement to prevent errors like
+            // "integer $integer" being converted to "int $int"
```
Comment (keradus) [inline Symfony/CS/Fixer/Contrib/PhpdocScalarFullFixer.php]: really? so much copy-paste...

**Comment by GrahamCampbell** [comment]
This fixer doesn't make sense since `integer` is actually wrong according to php 7.

File: Symfony/CS/Fixer/Contrib/PhpdocScalarFullFixer.php (hunk 1 above)
Comment (rokerkony) [inline Symfony/CS/Fixer/Contrib/PhpdocScalarFullFixer.php]: true... sorry for that. I will separate it to abstract parent

**Comment by rokerkony** [comment]
@GrahamCampbell according what i know about php 7 is nothing like making integer as deprecated or something like that... type casting is all different thing.
Can you provide me where I can find that integer in phpdoc is wrong?

**Comment by GrahamCampbell** [comment]
The official RFC says that the correct type is `int` and anything else is an alias.

It's considered a bad practice to use aliases, hence why php-cs-fixer has various alias fixers.

**Comment by rokerkony** [comment]
OK thanks, in this case I will close this PR.

**Pull request closed without merging**